src/index.test-d.ts

```diff
@@ -0,0 +1,47 @@
+import createDMG = require('.');
+
+createDMG({
+  appPath: '',
+  name: 'my dmg',
+}).then(() => {
+  // ...
+});
```

Comment:
Pretty sure `tsd` allows for top-level await:
```suggestion
await createDMG({
  appPath: '',
  name: 'my dmg',
});
```

Also, one of these `createDMG` calls should assert the appropriate return type.

Reply:
> Pretty sure tsd allows for top-level await:

Not needed when using `expectType`.